google/cloud/security/common/data_access/violation_dao.py

```diff
@@ -62,6 +62,10 @@ def insert_violations(self, violations, resource_name,
             # Create the violations snapshot table.
             snapshot_table = self._create_snapshot_table(
                 resource_name, snapshot_timestamp)
+        except MySQLdb.OperationalError, e:
```

Comment:
Oh, now I understand what's happening -- I just noticed that you are assembling the snapshot table name, and then using that in the map further down. Yes, please do put a TODO to check for the violations table. It would be nice not to see the log warning about the violations table existing already.